test/Chai-passport_test/b2c_oidc_hybrid_and_code_flow_test.js

```diff
@@ -0,0 +1,348 @@
+var id_token_in_token_resp = 'eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiIsImtpZCI6IklkVG9rZW5TaWduaW5nS2V5Q29udGFpbmVyLnYyIn0.eyJleHAiOjE0NzU3OTg4MTUsIm5iZiI6MTQ3NTc5NTIxNSwidmVyIjoiMS4wIiwiaXNzIjoiaHR0cHM6Ly9sb2dpbi5taWNyb3NvZnRvbmxpbmUuY29tLzIyYmY0MGM2LTExODYtNGVhNS1iNDliLTNkYzRlYzBmNTRlYi92Mi4wLyIsInN1YiI6Ik5vdCBzdXBwb3J0ZWQgY3VycmVudGx5LiBVc2Ugb2lkIGNsYWltLiIsImF1ZCI6ImYwYjZlNGViLTJkOGMtNDBiNi1iOWM2LWUyNmQxMDc0ODQ2ZCIsImFjciI6ImIyY18xX3NpZ25pbiIsIm5vbmNlIjoiWk04WVgySEN5Rm41ZFZ0NFFHQ3RKUWJaODBIWG9zL2QiLCJpYXQiOjE0NzU3OTUyMTUsImF1dGhfdGltZSI6MTQ3NTc5NTIxNSwib2lkIjoiNDMyOWQ2YmMtMGY4NC00NWQ4LTg3MDktMmM4YjA5MTM1N2QxIiwiZW1haWxzIjpbInNpanVuLndvcmtAZ21haWwuY29tIl0sImF0X2hhc2giOiJhRjVGV21DX1NlTGZ3VW5RQjZkbjRRIn0.Tm1sk9WKx6TbMxsCY0DFpqk8qIj9YgQLGCILvmFCeEaJwHOp85BVkfzcihSEzml1yqrH1dPLee2A3s22j9Si-sKSvq1VE9WpfXhvRuwP17snUc6vPn3ZESet47MowQMiY0uvVvvCg0vNCBaSgJSc58DxMZv6L_717Ysuuj5uUsh9sGuEpybo8tvKEPVsO2F7_R2t-8kM38XN4mUsvPUtBZhUHdu7bZ0jRcao5d3lAI_ZkgRCIgOfV9-Yu8A9Yj485HjENKZKmxTLg14__KSy6kfBVIbeH2DAYGZswo-ACGQpznCP47yE87zjWa5HSuKdrA7Iy7wwYu8ZFdGqUliTKg';
+var access_token = 'eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiIsImtpZCI6IklkVG9rZW5TaWduaW5nS2V5Q29udGFpbmVyLnYyIn0.eyJpc3MiOiJodHRwczovL2xvZ2luLm1pY3Jvc29mdG9ubGluZS5jb20vMjJiZjQwYzYtMTE4Ni00ZWE1LWI0OWItM2RjNGVjMGY1NGViL3YyLjAvIiwiZXhwIjoxNDc1Nzk4ODE1LCJuYmYiOjE0NzU3OTUyMTUsImF1ZCI6ImYwYjZlNGViLTJkOGMtNDBiNi1iOWM2LWUyNmQxMDc0ODQ2ZCIsIm9pZCI6IjQzMjlkNmJjLTBmODQtNDVkOC04NzA5LTJjOGIwOTEzNTdkMSIsInN1YiI6Ik5vdCBzdXBwb3J0ZWQgY3VycmVudGx5LiBVc2Ugb2lkIGNsYWltLiIsImVtYWlscyI6WyJzaWp1bi53b3JrQGdtYWlsLmNvbSJdLCJub25jZSI6IlpNOFlYMkhDeUZuNWRWdDRRR0N0SlFiWjgwSFhvcy9kIiwidmVyIjoiMS4wIn0.R4-nKaHb8GfvA1patSkVAfl3FqRw_-JPLzzktGfmQ8Fc0x9dIVMQovHleOMM2B4Y-jpyRWd-aCQ3g0GVldGJcPL2IiO0qj3jggZOKOn94hG85zUjFruc8LoT_qtkezhIk7_OOSUgbp-YZ-0URkP2OHnNM6Fzo8mfqOHQGdyGVzcTb-CERrFQdKFua6Br7Z8xqEEr0N3BVE12kA73uja3P9E_QB_hFlR-egDEoVgJ2mqYhSjPlMcB3D85tvMe3Wujx0qUNN8xQB6oVbp1aahD5swIkRaL3jvmc7uNjPsStSkTbcyg4g6ygh14TZ3JdnZ3zR5dUQjPdX7fkmiI3KCQHg';
+
+var PEMkey = "-----BEGIN RSA PUBLIC KEY-----\n\
+MIIBCgKCAQEAs4W7xjkQZP3OwG7PfRgcYKn8eRYXHiz1iK503fS+K2FZo+Ublwwa\n\
+2xFZWpsUU/jtoVCwIkaqZuo6xoKtlMYXXvfVHGuKBHEBVn8b8x/57BQWz1d0KdrN\n\
+XxuMvtFe6RzMqiMqzqZrzae4UqVCkYqcR9gQx66Ehq7hPmCxJCkg7ajo7fu6E7dP\n\
+d34KH2HSYRsaaEA/BcKTeb9H1XE/qEKjog68wUU9Ekfl3FBIRN+1Ah/BoktGFoXy\n\
+i/jt0+L0+gKcL1BLmUlGzMusvRbjI/0+qj+mc0utGdRjY+xIN2yBj8vl4DODO+wM\n\
+wfp+cqZbCd9TENyHaTb8iA27s+73L3ExOQIDAQAB\n\
+-----END RSA PUBLIC KEY-----\n";
+
+
+/*
+ * test strategy (for response_type = 'id_token') which checks the expiration of id_token
+ */
+
+var options = {
+  redirectUrl: 'https://localhost:3000/auth/openid/return',
+  clientID: 'f0b6e4eb-2d8c-40b6-b9c6-e26d1074846d',
+  identityMetadata: 'https://login.microsoftonline.com/mytenant.onmicrosoft.com/v2.0/.well-known/openid-configuration',
+  responseType: 'code id_token',
```

Comment:
I see you're testing the hybrid flow here. The PR indicates you're testing the code flow too. Where do you test responseType: 'code'?